### PR TITLE
Add common Terra.js examples. 

### PR DESCRIPTION
**Pull request by octalmage**
Here's a few common Terra.js examples to get this section started! 

localhost_8081_SDKs_Terra-js_Common-examples html (1)

I hope to add more as time goes on.

### CODE REVIEW

**Review by hanjukim — APPROVED** [review]

File: docs/SDKs/Terra-js/Common-examples.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Common examples
+
+## Configuring LCDClient
+
```
Comment (stevenTerra) [inline docs/SDKs/Terra-js/Common-examples.md:4]: The following code example shows how to initialize the LCDClient. The rest of the examples assume you initialized it by using this example or similar code.

File: docs/SDKs/Terra-js/Common-examples.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+const lcd = new LCDClient({
+  URL: "https://bombay-lcd.terra.dev/",
+  chainID: "bombay-12",
+  gasPrices: [new Coin("uluna", "0.01133")],
+  gasAdjustment: "1.5",
+  gas: 10000000,
+});
+```
+
+The rest of the examples assume LCDClient was initialized similar to the above example.
```
Comment (stevenTerra) [inline docs/SDKs/Terra-js/Common-examples.md]: I moved this phrase to the previous section. If you want to keep it here instead, let's say, "The following examples assume you initialized the LCDClient by using the previous code example or similar code."

File: docs/SDKs/Terra-js/Common-examples.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  gasPrices: [new Coin("uluna", "0.01133")],
+  gasAdjustment: "1.5",
+  gas: 10000000,
+});
+```
+
+The rest of the examples assume LCDClient was initialized similar to the above example. 
+
+## Sending native tokens
+
```
Comment (stevenTerra) [inline docs/SDKs/Terra-js/Common-examples.md:25]: The following code example shows how to send native tokens:

File: docs/SDKs/Terra-js/Common-examples.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+);
+
+const tx = await wallet.createAndSignTx({ msgs: [send] });
+const result = await lcd.tx.broadcast(tx);
+
+console.log(result);
+```
+
+## Sending CW20 tokens
+
```
Comment (stevenTerra) [inline docs/SDKs/Terra-js/Common-examples.md:53]: The following code example shows how to send CW20 tokens:

**Comment by octalmage** [comment]
@stevenTerra I made the changes and added a new example! Let me know what you think.

**Comment by octalmage** [comment]
Also updated the og screenshot.

**Comment by stevenTerra** [comment]
LGTM!

**Review by stevenTerra — APPROVED** [review]

**Pull request merged**